### PR TITLE
Docs: Updated main Link example

### PR DESCRIPTION
**Pull request by ponori**
Made the main Link example more clear that it was a Link example

### CODE REVIEW

File: docs/pages/web/link.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         description={generatedDocGen?.description}
         defaultCode={`
 <Text inline>
-  Find tips and best practices on the
-  <Link href="https://business.pinterest.com/" inline> Pinterest Business Site </Link>
+To see how you can grow your business, visit
+  <Link href="https://business.pinterest.com/advertise" inline externalLinkIcon="default" target="blank">Pinterest Ads</Link>
```
Comment (AlbertCarreras) [inline docs/pages/web/link.js:21]: technically, Pinterest links dont have the visit icon, just non-Pinterest urls.

Comment (AlbertCarreras) [reply docs/pages/web/link.js:21]: Screen Shot 2022-09-27 at 9 09 27 PM

Comment (ponori) [reply docs/pages/web/link.js:21]: Hmm... Good point.  Not to get too esoteric here, but would a Pinterest marketing site be considered leaving Pinterest content?

Comment (ponori) [reply docs/pages/web/link.js:21]: Seems like we're ok to move forward as it - https://pinterest.slack.com/archives/C014X9LTRCN/p1664388995735539

Comment (AlbertCarreras) [reply docs/pages/web/link.js:21]: Let's bring this to tomorrow's meeting as per thread

**Review by AlbertCarreras — APPROVED** [review]

**Pull request merged**